### ISSUE TITLE
bug(google-maps): ng update @angular/material does not update @angular/google-maps

### ISSUE THREAD

**Issue by Akxe** [angular/components#19947]
#### Reproduction

Steps to reproduce:
1. Have an Angular 9 app
2. Update to Angular 10
 
 
#### Expected Behavior

Either https://update.angular.io/#9.0:10.0l3 should inform that it needs to be updated manually too, or it should update with @angualr/material

#### Actual Behavior

Did only update @angular/material and @anuglar/ckd.

#### Environment

- Angular:
- CDK/Material: 9 => 10
- Browser(s): ---
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by crisbeto** [comment]
I'd say that this works as expected since `@anguar/google-maps` isn't related to `@angular/material` and `@angular/cdk`, apart from being in the same repository. I suppose we could provide an `ng update` schematic for the Google Maps package specifically.

**Comment by Akxe** [comment]
It wasn't really hard to upgrade it manually, but since there are breaking changes, I had to go through my code to fix them manually. Either an automatic update or a list of changes to look for after updating would be nice.

**Comment by devversion** [comment]
Yeah, it works as expected that `ng update @angular/material` does not update the youtube-player or google-maps package as those are separate unrelated packages. I agree though that we should probably set up update schematics for those packages if there are breaking changes.

**Comment by Akxe** [comment]
I can understand why `ng update @angular/material` does not update other, but the update.angular.io should have a note about updating these

**Comment by Splaktar** [comment]
@Akxe you can open an issue for update.angular.io here: https://github.com/StephenFluin/angular-update-guide/issues
